server/config.py

```diff
@@ -59,12 +47,7 @@
 
 max_requests_jitter = 5000
 
-
-def do_post_fork(server, worker):
-    patch_psycopg()
-
-
-post_fork = do_post_fork
```

Comment:
What's with the change in post_fork busines? Feel free to chat with me about this rather than responding here.

Reply:
It turns out we could not find any documentation suggesting why this was done 'post_fork'